build-eslint-command.js

```diff
@@ -0,0 +1,9 @@
+const shelljs = require(`shelljs`);
```

Comment:
You can disable eslint for a particular line using `// eslint-disable-next-line no-sync`. Just try to put a code comment above it for why you disabled it.
https://stackoverflow.com/questions/27732209/turning-off-eslint-rule-for-a-specific-line#29592334